rules/crowdstrike_rules/crowdstrike_suspicious_cron.py

```diff
@@ -0,0 +1,58 @@
+import shlex
+from fnmatch import fnmatch
+
+from panther_base_helpers import (
+    crowdstrike_detection_alert_context,
+    deep_get,
+    filter_crowdstrike_fdr_event_type,
+    get_crowdstrike_field,
+)
+
+SUSPICIOUS_CRON_CMD_ARGS = {
+    # Running in unexpected locations
+    "/tmp/*",  # nosec
+    # Reaching out to the internet
+    "curl",
+    "dig",
+    "http?://*",
+    "nc",
+    "wget",
+}
+
+SUSPICIOUS_CRON_CMDS = {
+    # Passing arguments into /bin/sh
+    "*|*sh",
+    "*sh -c *",
+}
+
+
+def suspicious_cmd_pairs(command):
+    return any((fnmatch(command, c) for c in SUSPICIOUS_CRON_CMDS))
+
+
+def suspicious_cmd_args(command):
+    command_args = shlex.split(command.replace("'", "\\'"))  # escape single quotes
```

Comment:
I don't think escaping is necessary here. Can you include a test case where you expect this is needed?